lib/pbench/server/templates.py

```diff
@@ -158,103 +249,90 @@ def __init__(
         self,
         prefix: str = None,
         mappings: Path = None,
-        settings: Path = None,
-        base: "TemplateFile" = None,
+        settings: JsonFile = None,
+        skeleton: JsonFile = None,
         tool: str = None,
-        skeleton: bool = False,
     ):
         """
         Describe an Elasticsearch template including a JSON document mappings
-        file, a JSON settings file, or both.
+        file and a JSON settings file, which will be merged into a full
+        Elasticsearch template payload.
 
-        A "base" template may also be linked, from which a final template will
-        be built using shared settings and optionally base mappings.
+        A "skeleton" JSON file may also be linked, from which the mappings will
+        be built by merging tool-specific properties and version into a common
+        base template.
 
         Args:
             prefix: Pbench index prefix string
             mappings: JSON document description. Defaults to None.
             settings: JSON index settings. Defaults to None.
-            base: Referenced base settings and/or mappings. Defaults to None.
             tool: Tool mapping files stitch together mapping JSON from the
-                main and base template objects using a parsed "tool" name.
-            skeleton: The JSON mappings file does not have a version.
+                main and skeleton templates using a parsed "tool" name.
+            skeleton: A versionless skeleton mappings file to merge with the tool
+                mappings.
```

Comment:
It would probably be better to have the descriptions in the same order as the function signature.

Reply:
Nevermind:  you fixed this in a subsequent commit.